searx/settings.yml

```diff
@@ -186,6 +186,12 @@ engines:
       require_api_key: false
       results: HTML
 
+  - name : artic
+    engine : artic
+    shortcut : arc
+    categories : general
```

Comment:
As I see, the engine defines the category "images" and here it rewrites it to general. I think it would be better to keep it in the images category. So, please remove this line if you agree. Otherwise it LGTM, thanks!